src/components/MenuTrigger/burger-menu.scss

```diff
@@ -0,0 +1,77 @@
+@import 'globals';
+@import './animations';
+
+$menu-breakpoint: map-get($breakpoints, 'navigation-menu');
+
+.menu-trigger-wrapper {
+  position: absolute;
+  top: 50%;
+  right: 0;
+  cursor: pointer;
+
+  @include media('>=#{$menu-breakpoint}') {
```

Comment:
the library allows you to just do
`@include media('>=menu-breakpoint') {`

it looks in `$breakpoints` automatically for the ref 😄